lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -172,6 +208,11 @@ defmodule Kernel.LexicalTracker do
   defp add_reference(d, module, :compile) when is_atom(module),
     do: :ets.insert(d, {{:mode, module}, :compile})
 
+  defp add_reference(d, module, fas, line, :runtime) when is_atom(module),
+    do: :ets.insert_new(d, {{:mode, module, fas, line}, :runtime})
```

Comment:
One thing I wanted to note here is that with the current table type of "set" we can't track multiple references to the same MFA on the same line. I figured this seemed like an acceptable compromise against additional complexity, but we could also create a second table that is purely for these references so that they can accumulate over a single line.